src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AnnotationUtils.java

```diff
@@ -36,7 +32,51 @@ public static String encodeValueList(final List<Double> valueList, final String
      * @return a comma-separated String
      */
     public static String encodeStringList( final List<String> stringList) {
-        return StringUtils.join(stringList, ",");
+        return StringUtils.join(stringList, LIST_DELIMITER);
+    }
+
+    /**
+     * Helper function to convert a List of Strings to a comma-separated String
+     * @param somethingList the ArrayList with String data
+     * @return a comma-separated String
+     */
+    public static String encodeAnyASList( final List<?> somethingList) {
+        return StringUtils.join(somethingList, ALLELE_SPECIFIC_PRINT_DELIM).replaceAll(BRACKET_REGEX, "");  //Who actually wants brackets at the ends of their string?  Who???
+    }
+
+    /**
+     * Helper function to determine if an annotation is allele-specific
+     * @param annotation the annotation to be tested
+     * @return true if the annotation is expected to have values per-allele
+     */
+    public static boolean isAlleleSpecific(final InfoFieldAnnotation annotation) {
```

Comment:
There is a danger that if someone adds a new allele-specific annotation one day, they will not realize that they have to update this method as well. Perhaps we could introduce an empty `AlleleSpecificAnnotation` marker interface, and have just these 4 classes implement it? That would be much less likely to be missed in the future...